### DISCUSSION TITLE
Frappe Framework Provider

### DISCUSSION THREAD

**Discussion by pipech** [refinedev/refine#5716] [General]
Hello everyone,

I just develop a new provider for the Frappe Framework and wanted to share it with you all!

For those who might not be familiar, the [Frappe Framework](https://frappeframework.com/) is this awesome open-source Low Code Framework that's behind [ERPNext](https://erpnext.com/). Super cool for building apps quickly.

Here's where you can check it out and dive into the code: [refine-frappe-provider on GitHub](https://github.com/pipech/refine-frappe-provider).

Would love to hear your thoughts or suggestions for enhancements.

**Comment by BatuhanW** [comment]
Hey @pipech thanks for sharing! It looks great. I think creating an example would be great way to show code and demo it.

**Comment by pipech** [comment]
Hi @BatuhanW,

A example app would be a base refine apps with few line added.
Setting up a Frappe instance and keeping it online might be quite costly in long run.
Are there any particular features or aspects you're keen on seeing on the demo?

```tsx
// App.tsx

import { Refine } from "@refinedev/core";

import {
  accessControlProvider,
  authProvider,
  dataProvider,
  type ClientParams,
} from "refine-frappe-provider";

const providerConfig = {
  url: 'https://frappe-server-url.com',
} satisfies ClientParams;

const App = () => (
  <Refine
    /* ... */
    accessControlProvider={
      accessControlProvider(providerConfig)
    }
    authProvider={
      authProvider(providerConfig)
    }
    dataProvider={
      dataProvider(providerConfig)
    }
  />
);
```
